internal/quaiapi/quai_api.go

```diff
@@ -766,12 +773,21 @@ func (s *PublicBlockChainQuaiAPI) RequestDomToAppendOrFetch(ctx context.Context,
 	}
 	s.b.RequestDomToAppendOrFetch(requestDom.Hash, requestDom.Entropy, requestDom.Order)
 }
-func (s *PublicBlockChainQuaiAPI) NewGenesisPendingHeader(ctx context.Context, raw json.RawMessage) {
-	var pendingHeader *types.Header
+
+type NewGenesisWorkObject struct {
+	WoHeader *types.WorkObjectHeader `json:"woHeader"`
+	WoBody   *types.WorkObjectBody   `json:"woBody"`
+
+	// TODO: needs to decode transaction
+	// Transaction *types.Transaction      `json:"transaction"`
```

Comment:
Dont we need this? it shouldnt crash now right since you are passing NewEmptyTx?

Reply:
fixed